news/279.bugfix

```diff
@@ -0,0 +1 @@
+Tuples weren't checked when creating a List container.
```

Comment:
A good news fragment will tell the user what bug was fixed , or what feature was added.
In this case, it should say something about Optional[Tuple] now being supported in DictConfig.